### PR TITLE
[tests-only] updated artifact upload location to `cache_public_s3`

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
move usage of `bucket: owncloud` to `cache_public_s3` bucket

## Related Issue
- Fixes https://github.com/owncloud/web/issues/6441

## How Has This Been Tested?

- :robot:

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternal2</strong> https://drone.owncloud.com/owncloud/web/22967/60/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalUsers-shareWithUsers_feature-L342.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by kiranparajuli589** [comment]
attempting to upload failed: https://drone.owncloud.com/owncloud/web/22967/60/17

File: .drone.star [hunk 1]
```diff
@@ -2561,37 +2576,20 @@ def stopBuild():
...
-                "from_secret": "cache_s3_endpoint",
-            },
-            "path_style": True,
-            "source": "%s/tests/acceptance/reports/screenshots/**/*" % dir["web"],
-            "strip_prefix": "%s/tests/acceptance/reports/screenshots" % dir["web"],
-            "target": "/web/screenshots/${DRONE_BUILD_NUMBER}",
-        },
-        "environment": {
-            "AWS_ACCESS_KEY_ID": {
-                "from_secret": "cache_s3_access_key",
```
Comment (kiranparajuli589) [inline .drone.star:2580]: is this env. still needed and should be updated or the access key from settings should just work :thinking:?

**Comment by kiranparajuli589** [comment]
the test doesn't fail :(

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/22975/53/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAcceptShares-acceptShares_feature-L99.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/22977/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace //owncloud/web/22977/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-22-03-44-19.zip 

#### npx playwright show-trace //owncloud/web/22977/tracing/alice-shares-file-to-brian-alice-2022-2-22-03-45-27.zip 

#### npx playwright show-trace //owncloud/web/22977/tracing/alice-shares-folder-with-file-to-brian-alice-2022-2-22-03-46-36.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/22981/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace //owncloud/web/22981/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-23-06-15-43.zip 

#### npx playwright show-trace //owncloud/web/22981/tracing/alice-shares-file-to-brian-alice-2022-2-23-06-16-58.zip 

#### npx playwright show-trace //owncloud/web/22981/tracing/alice-shares-folder-with-file-to-brian-alice-2022-2-23-06-18-14.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/22982/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https:////owncloud/web/22982/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-23-06-43-40.zip 

#### npx playwright show-trace https:////owncloud/web/22982/tracing/alice-shares-file-to-brian-alice-2022-2-23-06-44-50.zip 

#### npx playwright show-trace https:////owncloud/web/22982/tracing/alice-shares-folder-with-file-to-brian-alice-2022-2-23-06-46-01.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/22988/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https:////owncloud/web/22988/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-23-08-15-20.zip 

#### npx playwright show-trace https:////owncloud/web/22988/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2022-2-23-08-15-39.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/22988/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https:////owncloud/web/22988/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-23-08-16-42.zip 

#### npx playwright show-trace https:////owncloud/web/22988/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2022-2-23-08-17-01.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/22994/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://https://cache.owncloud.com//owncloud/web/22994/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-23-09-18-19.zip 

#### npx playwright show-trace https://https://cache.owncloud.com//owncloud/web/22994/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2022-2-23-09-18-33.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/22994/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://https://cache.owncloud.com//owncloud/web/22994/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-23-09-20-43.zip 

#### npx playwright show-trace https://https://cache.owncloud.com//owncloud/web/22994/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2022-2-23-09-21-04.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/22997/15/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAcceptShares-acceptShares_feature-L99.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/22997/53/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAcceptShares-acceptShares_feature-L99.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/23004/53/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAcceptShares-acceptShares_feature-L99.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/23007/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23007/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-23-11-07-58.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23007/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2022-2-23-11-08-18.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23007/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-carol-2022-2-23-11-08-24.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/23007/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23007/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-23-11-10-23.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23007/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2022-2-23-11-10-46.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23007/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-carol-2022-2-23-11-10-55.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23007/tracing/alice-shares-file-to-brian-alice-2022-2-23-11-12-23.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/23007/tracing/alice-shares-file-to-brian-brian-2022-2-23-11-12-37.zip 


</p></details>

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by phil-davis — APPROVED** [review]
LGTM

**Pull request merged**